Delta/Experimental Features/Features/AlternateAppIcons.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  AlternateAppIcons.swift
+//  Delta
+//
+//  Created by Chris Rittenhouse on 5/2/23.
+//  Copyright © 2023 LitRitt. All rights reserved.
+//
+
+import SwiftUI
+
+import DeltaFeatures
+
+enum AppIcon: String, CaseIterable, CustomStringConvertible, Identifiable
+{
+    case normal = "Default"
+    case inverted = "Inverted"
+    case gba4ios = "GBA4iOS"
+    case skin = "Controller Skin"
+    case pixelated = "Pixelated"
+    
+    var description: String {
+        return self.rawValue
+    }
+    
+    var id: String {
+        return self.rawValue
+    }
+    
+    var author: String {
+        switch self
+        {
+        case .normal: return "Paul Thorsen"
+        case .inverted, .gba4ios, .skin, .pixelated: return "LitRitt"
```

Comment:
```suggestion
        case .normal: return "Caroline Moore"
        case .gba4ios: return "Paul Thorsen"
        case .inverted, .skin, .pixelated: return "LitRitt"
```

Caroline made the default Delta icon, whereas Paul made the GBA4iOS icon :)